student_class.rb

```diff
@@ -0,0 +1,15 @@
+require './person_class'
+
+class Student < Person
+  attr_accessor :classroom
+
```

Comment:
- [ ] Dear @zbahati, for students we need to have a method that serves as the 'belongs-to' side. Hence we need to create a method that takes a classroom and adds the student to the classroom unless the classroom already has that student. 
 **I see you put this logic inside the constructor, but there might be a chance the student is created without a classroom at first.**
> Something like 
```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```